demo/src/main/java/com/midtrans/demo/AppUtils.java

```diff
@@ -102,6 +104,10 @@ public static PaymentMethodsModel getVaPaymentMethods(Context context, String ty
                     model = new PaymentMethodsModel(PAYMENT_NAME_BNI_VA, null, 0, "", 5, status);
                     break;
 
+                case PaymentType.BRI_VA:
+                    model = new PaymentMethodsModel(PAYMENT_NAME_BRI_VA, null, 0, "", 5, status);
```

Comment:
Is the priority same with BNI?

Reply:
updated the priority please re-check @farhanpahlevi